### PR TITLE
[cs] correct invalid @param types

### PR DESCRIPTION
**Pull request by TomasVotruba**
| Q             | A
| ------------- | ---
| Branch?       | master
| Bug fix?      | no
| New feature?  | no
| BC breaks?    | no
| Deprecations? | no
| Tests pass?   | yes
| Fixed tickets | -
| License       | MIT
| Doc PR        | -

Correcting few @param annotations.

Running https://github.com/Symplify/Symplify/pull/1206

### CODE REVIEW

**Review by DAcodedBEAT — COMMENTED** [review]
Thank you @TomasVotruba for working on this!  I am always a fan of correct documentation of code so this PR was warmly welcomed 👍I addressed some potential typos and possible additions in the comments below.

File: src/Symfony/Component/Routing/Tests/Matcher/Dumper/PhpMatcherDumperTest.php [hunk 1]
```diff
@@ -473,7 +473,7 @@ public function getRouteCollections()
     }
 
     /**
-     * @param $dumper
+     * @param $collection
```
[the remark this answers is not in the document]
Comment (TomasVotruba) [reply src/Symfony/Component/Routing/Tests/Matcher/Dumper/PhpMatcherDumperTest.php]: This PR only aims on parameter format corrections.

Adding missing ones or even typehints is for another job.

File: src/Symfony/Component/VarDumper/Caster/ClassStub.php [hunk 2]
```diff
@@ -21,8 +21,8 @@
 class ClassStub extends ConstStub
 {
     /**
-     * @param string   A PHP identifier, e.g. a class, method, interface, etc. name
-     * @param callable The callable targeted by the identifier when it is ambiguous or not a real PHP identifier
+     * @param string   $identifier A PHP identifier, e.g. a class, method, interface, etc. name
+     * @param callable $callable   The callable targeted by the identifier when it is ambiguous or not a real PHP identifier
```
[the remark this answers is not in the document]
Comment (TomasVotruba) [reply src/Symfony/Component/VarDumper/Caster/ClassStub.php]: As above-mentioned

File: src/Symfony/Component/VarDumper/Caster/RedisCaster.php [hunk 3]
```diff
@@ -101,7 +101,7 @@ public static function castRedisCluster(\RedisCluster $c, array $a, Stub $stub,
     }
 
     /**
-     * @param \Redis|\RedisArray|\RedisCluster
+     * @param \Redis|\RedisArray|\RedisCluster $redis
```
[the remark this answers is not in the document]
Comment (TomasVotruba) [reply src/Symfony/Component/VarDumper/Caster/RedisCaster.php]: As above-mentioned

File: src/Symfony/Component/Routing/Tests/Matcher/Dumper/PhpMatcherDumperTest.php (hunk 1 above)
[the remark this answers is not in the document]
Comment (fabpot) [reply src/Symfony/Component/Routing/Tests/Matcher/Dumper/PhpMatcherDumperTest.php]: Actually, the right fix here is to remove this `@param` as it does not bring any value (no typehint and no description).

File: src/Symfony/Component/VarDumper/Caster/RedisCaster.php (hunk 3 above)
[the remark this answers is not in the document]
Comment (fabpot) [reply src/Symfony/Component/VarDumper/Caster/RedisCaster.php]: FYI, we do not want to have *all* `@param`, only the ones that add value (which is the case here for the first arg).

[the remark this answers is not in the document]
Comment (TomasVotruba) [reply src/Symfony/Component/VarDumper/Caster/RedisCaster.php]: I know. I've tried cleanup those before but it was closed without further feedback. I want this to merge, so I keep it small :)

File: src/Symfony/Component/DependencyInjection/Dumper/PhpDumper.php [hunk 4]
```diff
@@ -1250,7 +1250,7 @@ public function getParameterBag()
     /*{$this->docStar}
      * Computes a dynamic parameter.
      *
-     * @param string The name of the dynamic parameter to load
+     * @param string \$name The name of the dynamic parameter to load
```
[the remark this answers is not in the document]
Comment (TomasVotruba) [reply src/Symfony/Component/DependencyInjection/Dumper/PhpDumper.php]: Here it's needed, so it's not interpreted as variable, but as string. NOWDOW

**Comment by TomasVotruba** [comment]
@DAcodedBEAT I've resolved your comment in the scope of this PR.

I also added one more change with @return $values. What do you think about it?

(No more changes are planned in this PR to keep it small. Other changes will follow in separated PR.)

File: src/Symfony/Component/HttpKernel/HttpCache/HttpCache.php [hunk 5]
```diff
@@ -1,5 +1,14 @@
 <?php
 
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
```
Comment (TomasVotruba) [inline src/Symfony/Component/HttpKernel/HttpCache/HttpCache.php]: This was required by fabpot.io to pass, not sure about it.

Comment (nicolas-grekas) [reply src/Symfony/Component/HttpKernel/HttpCache/HttpCache.php]: fabbot has some false-positives, this is one of them :)

File: src/Symfony/Component/DependencyInjection/Tests/Fixtures/php/services_json_env.php [hunk 6]
```diff
@@ -94,7 +94,7 @@ public function getParameterBag()
     /**
      * Computes a dynamic parameter.
      *
-     * @param string The name of the dynamic parameter to load
+     * @param string $name The name of the dynamic parameter to load
```
[the remark this answers is not in the document]
Comment (nicolas-grekas) [reply src/Symfony/Component/DependencyInjection/Tests/Fixtures/php/services_json_env.php]: This change doesn't apply to branch 2.8, isn't it? I applied it to upper ones.

[the remark this answers is not in the document]
Comment (TomasVotruba) [reply src/Symfony/Component/DependencyInjection/Tests/Fixtures/php/services_json_env.php]: I don't how this merging to 2.8 and then merging to upper branches works. I thought it was lost.

I see it here now: https://github.com/symfony/symfony/commit/b634ed564a0e8ad5d23457a89fa9267d85112345#diff-1cbd9d5b08f0aec84c777295d4b11bc3

[the remark this answers is not in the document]
Comment (nicolas-grekas) [reply src/Symfony/Component/DependencyInjection/Tests/Fixtures/php/services_json_env.php]: It works by hand :) We merge on the lowest branch where it applies, then we merge manually lower branches to upper ones.

[the remark this answers is not in the document]
Comment (TomasVotruba) [reply src/Symfony/Component/DependencyInjection/Tests/Fixtures/php/services_json_env.php]: But how is possible, that the change that I linked is not in lower branch but appears in upper one?

It got hidden somewhere? :)

**Pull request closed without merging**